pkg/sriov/network.go

```diff
@@ -90,6 +90,25 @@ func (builder *NetworkBuilder) WithVLAN(vlanID uint16) *NetworkBuilder {
 	return builder
 }
 
+// WithQinQSupport sets the VLAN protocol for qinq tunneling protocol in the SrIovNetwork definition spec.
+func (builder *NetworkBuilder) WithQinQSupport(vlanProtocol string) *NetworkBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	if vlanProtocol == "" {
+		builder.errorMsg = "SrIovNetwork 'vlanProtocol' cannot be empty"
+	}
+
+	if builder.errorMsg != "" {
```

Comment:
```
	if builder.errorMsg != "" {
		return builder
	}
```
Remove this block and instead add "return builder" in line 101. Just to avoid the error checking when we are already sure of it.

Reply:
removed